src/en/juju-upgrade.md

```diff
@@ -0,0 +1,191 @@
+Title: Upgrading Juju software
+
+
+# Upgrading Juju software
+
+At the highest level, a Juju topology can be divided into two main parts:
+
+- Juju client
```

Comment:
would it be worth explaining what you mean by this? e.g. "Juju client -  (the software you run locally to control Juju deployments...